alacritty_terminal/src/tty/unix.rs

```diff
@@ -148,6 +150,15 @@ pub fn new<C>(config: &Config<C>, size: &SizeInfo, window_id: Option<usize>) ->
 
     let (master, slave) = make_pty(win_size);
 
+    #[cfg(any(target_os = "linux", target_os = "macos"))]
+    {
+        if let Ok(mut termios) = termios::tcgetattr(master) {
+            // Assume UTF-8 encoded characters.
```

Comment:
I think `assume` is a bit misleading here, since UTF-8 is basically all Alacritty does.

We know we want UTF-8, we just need to tell that to the kernel.

```suggestion
            // Set character encoding to UTF-8.
```